core/templates/services/keyboard-shortcut.service.ts

```diff
@@ -73,6 +73,22 @@ export class KeyboardShortcutService {
     Mousetrap.bind('?', () => {
       this.openQuickReference();
     });
+
+    Mousetrap.bind('left', () => {
+      var previousButton = document.querySelector(
```

Comment:
try to use let on the place of var if possible.

Reply:
@pankajdahiya008 Can you please replace it now ? It's not much of a work, also it's always better to follow best practises.

Reply:
Done